.github/workflows/run-test.yml

```diff
@@ -66,6 +66,7 @@ jobs:
             */build/reports/jacoco/codeCoverage/html/**
 
       - name: Add coverage to PR
+        if: always()
```

Comment:
info: I don't know if this will work, but you probably need to just merge it to ensure it works :)

I think you can try also: `if: ${{ github.event_name != 'schedule' }}